File: src/test/java/test/ListOrganizationsTest.java
package test
import cn.authing.sdk.java.client.ManagementClient;
import cn.authing.sdk.java.dto.*;
import cn.authing.sdk.java.model.ManagementClientOptions;
import cn.authing.sdk.java.util.JsonUtils;

import java.util.Collections;



public class ListOrganizationsTest {

    private static final String ACCESS_KEY_ID = "YOUR_ACCESS_KEY_ID";
    private static final String ACCESS_KEY_SECRET = "YOUR_ACCESS_KEY_SECRET";


    public static void main(String[] args) throws Throwable {
        ManagementClientOptions clientOptions = new ManagementClientOptions(ACCESS_KEY_ID, ACCESS_KEY_SECRET);
        ManagementClient client = new ManagementClient(clientOptions);

        ListOrganizationsDto request = new ListOrganizationsDto();
        request.setPage(0);
        request.setLimit(0);
        request.setFetchAll(Boolean.TRUE);
        request.setWithCustomData(Boolean.TRUE);

        OrganizationPaginatedRespDto response = client.listOrganizations(request);
        System.out.println(JsonUtils.serialize(response));
    }

}

File: src/test/go/test/ListOrganizationsTest.go
package main

import (
"authing-go-sdk/client"
"authing-go-sdk/dto"
"fmt"
)

func main() {
    options := client.ManagementClientOptions{
        AccessKeyId:     "AUTHING_USERPOOL_ID",
        AccessKeySecret: "AUTHING_USERPOOL_SECRET",
    }
    var err error
    client, err := client.NewClient(&options)
    if err != nil {
        panic(err)
    }
    request := dto.ListOrganizationsDto{
        Page: 0,
        Limit: 0,
        FetchAll: false,
        WithCustomData: false,
    }
    response := client.ListOrganizations(&request)
    fmt.Println(response)
}

